docker/importer/importer.py

```diff
@@ -55,6 +57,22 @@ def _yaml_str_representer(dumper, data):
   return dumper.represent_scalar('tag:yaml.org,2002:str', data)
 
 
+def _repo_path(repo):
+  """Return local disk path to repo."""
+  # Remove '.git' component.
+  return os.path.dirname(repo.path.rstrip('/'))
```

Comment:
Is this rstrip needed even though we are using dirname?

Reply:
Yeah, pygit's Repository.path gives us `/path/.git/` and we need to strip it or we'll still have the `.git` in our path.